torch/__init__.py

```diff
@@ -345,20 +345,20 @@ def set_default_dtype(d):
     """
     _C._set_default_dtype(d)
 
-def use_deterministic_algorithms(d):
+def use_deterministic_algorithms(mode):
     r""" Sets whether PyTorch operations must use "deterministic"
     algorithms. That is, algorithms which, given the same input, and when
     run on the same software and hardware, always produce the same output.
-    When True, operations will use deterministic algorithms when available,
+    When enabled, operations will use deterministic algorithms when available,
```

Comment:
I figured "enabled" makes more sense since we're not explicitly referencing `mode` here